.github/workflows/manubot.yaml

```diff
@@ -47,6 +56,8 @@ jobs:
           echo "BUILD_PDF=${{ github.event.inputs.BUILD_PDF }}" >> $GITHUB_ENV
           echo "BUILD_DOCX=${{ github.event.inputs.BUILD_DOCX }}" >> $GITHUB_ENV
           echo "BUILD_LATEX=${{ github.event.inputs.BUILD_LATEX }}" >> $GITHUB_ENV
+          echo "SPELLCHECK=${{ github.event.inputs.SPELLCHECK }}" >> $GITHUB_ENV
```

Comment:
Confirmed that this overwrites `jobs.manubot.env.SPELLCHECK` in https://github.com/dhimmel/manubot-rootstock/runs/4515234391?check_suite_focus=true#step:4:12